Makefile

```diff
@@ -54,16 +54,30 @@ all: binaries manpages
 help:
 	@echo  '  binaries        - Build all binaries'
 	@echo  '  manpages        - Build manual pages'
+	@echo
+	@echo  '  limactl         - Build limactl, and lima'
+	@echo  '  lima            - Copy lima, and lima.bat'
+	@echo  '  helpers         - Copy nerdctl.lima, apptainer.lima, docker.lima, podman.lima, kubectl.lima'
+	@echo
+	@echo  '  guestagents               - Build guestagents for archs enabled by CONFIG_GUESTAGENT_ARCHS_*'
+	@echo  '  native-guestagent         - Build guestagent for native arch'
+	@echo  '  additional-guestagents    - Build guestagents for archs other than native arch'
+	@echo
+	@echo  '  templates                 - Copy templates'
+	@echo  '  template_experimentals    - Copy experimental templates'
+	@echo  '  default_template          - Copy default.yaml template'
+	@echo  '  create-examples-link      - Create a symlink at _output/share/lima/examples pointing to _output/share/lima/templates'
+	@echo
+	@echo  '  documents                 - Copy documents'
+	@echo  '  create-links-in-documents - Create symlinks in documents'
+	@echo
+	@echo  '  e.g. to install limactl, native guestagent, and templates:'
+	@echo  '   make clean limactl native-guestagent templates install'
```

Comment:
Can we have a custom target for this, so I can just run `make local`? 😄

Reply:
added `native` target